### PR TITLE
docs: GitHub Pull Request Updater plugin doc

### LINKED ISSUE #16: DOC: Create a documentation for GitHub Pull Request Updater plugin
Documentation for the GitHub Pull Request Updater plugin is not present in docs, it needs to be created and added in docs.

### PR DESCRIPTION
**Pull request by bhushan-nemade-dt**
Fixes #16

### CODE REVIEW

**Review by abhibhaw — DISMISSED** [review]

File: docs/user-guide/plugins/github-pull-request-updater.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# GitHub Pull Request Updater
+
+## Introduction
+The **GitHub Pull Request Updater** plugin automatically adds pipeline status and deployment information as comments on your GitHub pull requests. Every time your Devtron CI pipeline runs, you get instant visibility onto the specific pull request on GitHub. This seamless integration eliminates the need to switch between multiple tools or platforms to track your deployment, Development teams also saves their time by avoiding manual status updates and reducing the back-and-forth communication about deployment states. 
+
+### Prerequisites
+Before integrating the **GitHub Pull Request Updater** plugin, ensure you have an GitHub Personal Access Token (PAT) with appropriate permissions to access the your GitHub repository.
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/github-pull-request-updater.md]: (Replace 'an' with 'a' in the following line): `have an GitHub Personal Access Token`
(Remove 'the' in the following line): `access the your GitHub repository`

File: docs/user-guide/plugins/github-pull-request-updater.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# GitHub Pull Request Updater
+
+## Introduction
+The **GitHub Pull Request Updater** plugin automatically adds pipeline status and deployment information as comments on your GitHub pull requests. Every time your Devtron CI pipeline runs, you get instant visibility onto the specific pull request on GitHub. This seamless integration eliminates the need to switch between multiple tools or platforms to track your deployment, Development teams also saves their time by avoiding manual status updates and reducing the back-and-forth communication about deployment states.
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/github-pull-request-updater.md]: Change `deployment, Development teams also saves their` to `deployment. Development teams also save their`
(Replace comma with full-stop else it becomes a long sentence to read, and replace 'saves' with 'save')

File: docs/user-guide/plugins/github-pull-request-updater.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Add a brief explanation of the task and the reason for choosing the plugin. Include information for someone else to understand the purpose of the task.
+
+e.g., `A plugin to update GitHub pull request`
+
+### Input Variables
+
+| Variable                 | Format       | Description | Sample Value |
+| ------------------------ | ------------ | ----------- | ------------ |
+|   AccessToken             | STRING      | GitHub Personal Access Token (PAT) with appropriate permissions, used for authentication.  | ghp_1234abcd5678efgh9012ijkl3456mnop            |
+|   UpdateWithDockerImageI  | BOOL       | When set to true, the plugin will include Docker image information in PR comments. Default: True        |  true |
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/github-pull-request-updater.md]: It should be `UpdateWithDockerImageId` instead of `UpdateWithDockerImageI`

**Review by ashoknayak777 — CHANGES_REQUESTED** [review]
@bhushan-nemade-dt, added a few things to fix. Pls check.

File: docs/user-guide/plugins/github-pull-request-updater.md (hunk 1 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/github-pull-request-updater.md]: done

File: docs/user-guide/plugins/github-pull-request-updater.md (hunk 2 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/github-pull-request-updater.md]: done

File: docs/user-guide/plugins/github-pull-request-updater.md (hunk 3 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/github-pull-request-updater.md]: done

**Review by ashoknayak777 — DISMISSED** [review]
LGTM

**Review by abhibhaw — DISMISSED** [review]
LGTM

**Review by ashoknayak777 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]

**Pull request merged**